### ISSUE TITLE
Get data for all existing water temperature observations

### ISSUE THREAD

**Issue by jordansread** [GLEON/MilleLacsMystery#1]

**Comment by gjahansen2** [comment]
![volunteer](https://cloud.githubusercontent.com/assets/19436778/24306550/f70c6032-108e-11e7-8d38-0d3f7e7b7ac2.jpg)

**Comment by gjahansen2** [comment]
that's how you officially assign yourself to a task, right?

**Comment by lawinslow** [comment]
Yeah, that's how Github works.

**Comment by gjahansen2** [comment]
@jread-usgs @lawinslow I am compiling these data now, and I can see that all the validation data we had for the last run came from a single source. Do you want me to compile everything (including the temperature data we already have) into a single file?

**Comment by lawinslow** [comment]
@gjahansen2 hmm, you mean the data we got from Water Quality Portal is just a single source-worth of data, and that there are a lot more? For my organization, it wouldn't hurt to have the non-WQP data put up here. Then we could easily merge it with WQP-available data and not have overlap. (let me know if I misunderstood the question).

**Comment by gjahansen2** [comment]
Yes. all the data in the WQP appear to have come from MN PCA. MN DNR (fisheries) collects quite a bit of temperature data as well, including continuous loggers since 2000 (which were not included in the last set of validation data). So I could make one file that you don't have to merge that has 1. All temperature data from DNR, and 2. The most up to date PCA data (directly from them, not from WQP), in which case you would NOT merge with WQP data. Alternatively, I can just give you DNR data.

**Comment by lawinslow** [comment]
Hmm, maybe a weird way to respond, but could I have it all (for all lakes)? Then we could integrate it into the overall "data assimilation" pipeline and then just subset out the data we need for this lake.

**Comment by lawinslow** [comment]
Plus, it sounds like an awesome dataset.

**Comment by gjahansen2** [comment]
Sure. Right now I have pulled all Mille Lacs data together from loggers, DNR, and PCA into one file. This will overlap some with what you already have from the WQP so I don't think this is actually what you want. But it is everything we have for Mille Lacs. Note that in the previous validation file for Mille Lacs we had about 950 records - here we have almost 15,000.

[ML_observed_temperatures.txt](https://github.com/GLEON/MilleLacsMystery/files/873748/ML_observed_temperatures.txt)

Here is all the temperature data for all lakes from MDNR Fisheries database (excluding temperature loggers).
[MN_fisheries_all_temp_data.txt](https://github.com/GLEON/MilleLacsMystery/files/873751/MN_fisheries_all_temp_data.txt)

I think what you have from the WQP will pretty much be the same as what I can get from PCA (Still waiting on that).

I am also learning that a lot of temperature data exist outside our main databases, and I am slowly compiling some of that in the hopes of validating the temperature trends our model predicts (especially the large warming trends near Lake Superior). Those data are not ready yet but I am getting them together (perhaps worthy of a different Github project...)

**Comment by gjahansen2** [comment]
I have all the PCA temperature data (probably you have most of it already from the WQP). It is too big to add to Github.  It is here: https://www.dropbox.com/s/59opmklx8ssn3j1/MPCA_temp_data_all.txt?dl=0

I am not sure if that link will work since I am getting a warning that my Dropbox is full. @lawinslow let me know if you can grab those.
